oap-server/server-starter/src/main/resources/ui-initialized-templates/faas/faas-root.json

```diff
@@ -0,0 +1,239 @@
+                      "condition": ">",
+                      "value": "1"
+                    }
+                  ],
+                  "description": {
+                    "healthy": "Healthy",
+                    "unhealthy": "Success Rate < 95% and Traffic > 1 calls / min"
+                  },
+                  "nodeMetricConfig": [
+                    {
+                      "unit": "calls / min",
+                      "label": "Load"
+                    },
+                    {
+                      "calculation": "percentage",
+                      "unit": "%",
+                      "label": "Success Rate"
+                    },
+                    {
+                      "unit": "ms",
+                      "label": "Latency"
+                    }
+                  ],
+                  "linkServerMetricConfig": [
+                    {
+                      "unit": "ms",
+                      "label": "Server Latency"
+                    },
+                    {
+                      "unit": "calls / min",
+                      "label": "Server Load"
+                    }
+                  ],
+                  "linkClientMetricConfig": [
+                    {
+                      "unit": "calls / min",
+                      "label": "Client Load"
+                    },
+                    {
+                      "unit": "ms",
+                      "label": "Client Latency"
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Trace",
+              "children": [
+                {
+                  "x": 0,
+                  "y": 0,
+                  "w": 24,
+                  "h": 49,
+                  "i": "0",
+                  "type": "Trace",
+                  "widget": {
+                    "title": "Title"
+                  },
+                  "graph": {},
+                  "standard": {},
+                  "metrics": [
+                    ""
+                  ],
+                  "metricTypes": [
+                    ""
+                  ],
+                  "moved": false
+                }
+              ]
+            }
+          ],
+          "moved": false
+        },
+        {
+          "x": 0,
+          "y": 0,
+          "w": 24,
+          "h": 2,
+          "i": "2",
+          "type": "Text",
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "black",
+            "backgroundColor": "white",
+            "content": "FaaS (Function-as-a-Service) is a type of cloud-computing service that allows you to execute code in response to events without the complex infrastructure typically associated with building and launching microservices applications. OpenFunction as a FaaS platform, provides out-of-box observability with SkyWalking integration.",
```

Comment:
@FeynmanZhou Do you have any suggestions about this statement? And we are linking to your GitHub path, please take a look.

Reply:
LGTM

Reply:
As for the document link, I will inform you later. Now, this doc is under review and you could preview it [here](https://deploy-preview-54--openfunction-dev.netlify.app/docs/best-practices/skywalking-solution-for-openfunction/).

I could raise a PR to modify it to the OpenFunction website.

Reply:
Is there a pull request I could comment on? I would like to suggest `SkyWalking v9 required`

Reply:
> Use SkyWalking for OpenFunction as a Distributed Tracing Solution

Change to

> Use SkyWalking for OpenFunction as an observability solution.

Reply:
Good catch, the screenshot should be updated to SkyWalking v9, we will update it later. You can find the PR [here](https://github.com/OpenFunction/website/pull/54/files#diff-af2fbd3d6d086361a2eee05c4e3f117fc9ece85531c298c20b989ae70a242352).